src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,56 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }, { key }) => {
```

Comment:
```suggestion
export const MovieCard: React.FC<Props> = ({ movie }, { key }) => {
```
why do you need key here?

Reply:
Done